administrator/language/de-DE/plg_multifactorauth_webauthn.ini

```
@@ -0,0 +1,24 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_WEBAUTHN="Multi-Faktor-Authentifizierung - Web-Authentifizierung (WebAuthn)"
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_CREATE_INVALID_LOGIN_REQUEST="Ungültige Authentifizierungsanfrage."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_CREATE_INVALID_PK="Die Authentifikator-Registrierung ist fehlgeschlagen. Die vom Browser empfangene Authentifikator-Antwort stimmt nicht mit dem vom Server ausgegebenen öffentlichen Schlüssel überein. Dies bedeutet, dass jemand versucht hat, dies zu hacken, oder dass etwas defekt ist."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_CREATE_INVALID_USER="Aus Sicherheitsgründen ist es nicht erlaubt, Authentifikatoren im Namen eines anderen Benutzers zu registrieren."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_CREATE_NO_ATTESTED_DATA="Es ist etwas schief gelaufen, aber im Moment sind keine weiteren Informationen über den Fehler verfügbar. Bitte versuchen, den Authentifikator erneut zu registrieren."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_CREATE_NO_PK="Der Server hat keinen öffentlichen Schlüssel für die Authentifikatorregistrierung ausgegeben, aber irgendwie eine Authentifikatorregistrierungsanfrage vom Browser erhalten. Das bedeutet, dass jemand versucht hat, dies zu hacken, oder dass etwas defekt ist."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_NOTAVAILABLE_BODY="Der Browser unterstützt den WebAuthn-Standard nicht. Noch sind nicht alle Browser mit WebAuthn auf allen Geräten kompatibel."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_NOTAVAILABLE_HEAD="Der Browser bietet keine Unterstützung für WebAuthn"
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_NOTHTTPS_BODY="Bitte die Website über HTTPS aufrufen, um die Multi-Faktor-Authentifizierung mit WebAuthn zu aktivieren."
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_NOTHTTPS_HEAD="WebAuthn ist nur über HTTPS verfügbar"
+PLG_MULTIFACTORAUTH_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Es wurde noch kein Authentifikator konfiguriert oder der Authentifikator, welcher verwendet werden soll, ist nicht zulässig."
+PLG_MULTIFACTORAUTH_WEBAUTHN_LBL_CONFIGURED="Dieser Authentifikator ist bereits konfiguriert. Bitte beachten, dass auf dieser Seite nur der Titel geändert werden kann."
+PLG_MULTIFACTORAUTH_WEBAUTHN_LBL_DISPLAYEDAS="Web-Authentifizierung"
+PLG_MULTIFACTORAUTH_WEBAUTHN_LBL_INSTRUCTIONS="Die Schaltfläche &ldquo;%s&rdquo; auf dieser Seite verwenden, um den Web-Authentifizierungsprozess zu starten. Danach bitte den Anweisungen des Browsers folgende, um die Web-Authentifizierung mit dem bevorzugten Authentifikator abzuschließen."
```

```suggestion
PLG_MULTIFACTORAUTH_WEBAUTHN_LBL_INSTRUCTIONS="Die Schaltfläche &bdquo;%s&ldquo; auf dieser Seite verwenden, um den Web-Authentifizierungsprozess zu starten. Danach bitte den Anweisungen des Browsers folgen, um die Web-Authentifizierung mit dem bevorzugten Authentifikator abzuschließen."
```

Gibt es einen Grund, warum im Core die Anführungszeichen im HTML-Code eingefügt wurden?

---

Um es kenntlich zu machen.